usaspending_api/references/tests/test_load_naics.py

```diff
@@ -14,8 +14,12 @@ def test_naics_existing_and_new_files():
 
     call_command('load_naics')
 
-    naics_2002_data_total = NAICS.objects.all().filter(year=2002).count()
+    naics_cascade_count = (NAICS.objects.all().filter(year=2017).count()
```

Comment:
I recommend not adding all of the SQL search results together. You can have an `assert` for one or two years.

There are other test cases you could add if you want. Some ideas are checking there are zero rows in the model before the `load_naics` command is executed; searching for a specific NAICS, verifying all rows have years only in 2002,2007,2012,2017.